javascript-carousel/main.js

```diff
@@ -0,0 +1,68 @@
+/* const images = [
+    'images/001.png',
+    'images/025.png',
+    'images/039.png',
+    'images/004.png',
+    'images/007.png',
+ ];
+ */
+const images = document.querySelectorAll('img');
+let i = 1;
+
+// Auto carousel
+const interID = setInterval(function carousel() {
+  images.forEach((img) => {
+    img.className = 'hidden';
+  });
+  if (i === images.length) {
+    i = 0;
+  }
+  images[i].className = '';
+  i++;
+}, 3000);
+
+// Next img
+const nextBtn = document.querySelector('.fa-chevron-right');
+function handleNext(event) {
+  clearInterval(interID);
+  images.forEach((img) => {
+    img.className = 'hidden';
+  });
+  if (i === images.length) {
+    i = 0;
+  }
+  images[i].className = '';
+  i++;
+  setInterval(); // tried restarting interval again after incrementing.
```

Comment:
`setInterval` needs to be called with a callback function that runs at each tick. From the looks of what you have, that's the `carousel` function defined above.